docs/v2_migration.md

```diff
@@ -310,7 +322,7 @@ img = rio_tiler.utils.render(tile, mask, img_format="PNG")
 Each `rio_tiler.io.{mission}` **scene id parser** (e.g cbers_parser) has been refactored and now return AWS S3 path information.
 
 ```python
-rio_tiler.io.landsat8.landsat_parser("LC08_L1TP_016037_20170813_20170814_01_RT")) 
+rio_tiler.io.landsat8.landsat_parser("LC08_L1TP_016037_20170813_20170814_01_RT"))
```

Comment:
We can remove this section or just add a note about the new rio-tiler-pds

Reply:
Yeah I was going to leave `rio-tiler-pds` for the next docs PR